cloudseption/profile-app
# Adding user API with persistent storage using mongodb

Steps to test:
A **.env** file must be added to the root directory in order for this to work. Contact Cameron for what needs to be in this file.

Once you have run **npm install** and run the server again with **npm run dev**...

Open postman and make a GET request to **localhost:3000/users**
You should see a list of users.

Make a POST request with to **localhost:3000/users** with the following json body
`{
	"name" : "Patrick Charie",
	"description": "The awesome guy testing the user API."
}`

Make a GET request with **localhost:3000/users/5beb7ac6ba5eb8b771634913**, replacing the id with a valid id that you got from the previous get request.

Make a patch request with **localhost:3000/users/5beb7441e8facdb76ac84724**, replacing the id with a valid id that you got from the previous get request. Set the body to an array of desired property changes. For instance, if you want to change the name, set the body to
`[
	{"propName":"name", "value" : "Timmothy Blunder"}
]`
